src/Explorer/Explorer.tsx

```diff
@@ -2273,6 +2277,32 @@ export default class Explorer {
     );
   }
 
+  public openAddTableEntityPanel(queryTablesTab: QueryTablesTab, tableEntityListViewModel: TableListViewModal): void {
+    false
```

Comment:
This should be removed. It will always take the false path

Reply:
Removed